docs/public/replay-viewer-app/style.css

```
@@ -749,6 +749,11 @@ button:disabled {
   box-shadow: 0 0 12px rgb(255 100 124 / 40%);
 }
 
+.legend-swatch--skip {
```

Stylelint (17.14.0)

[error] Expected class selector ".legend-swatch--skip" to be kebab-case (selector-class-pattern)

(selector-class-pattern)